docs/tempo/website/operations/_index.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: Operations
+title: Deployment and Operations
```

Comment:
I kinda prefer having `Operations` as the title. (for me) Deployment is just a subsection inside Operations.

In the same way, that on the repo we have the Deployment stuff inside the [Operations](https://github.com/grafana/tempo/tree/main/operations) folder.

Reply:
So the intent of this was to just remove these single page top level items to reduce confusion and I felt like both Architecture and Deployment made sense under Operations. 

The reason for the title change to "Deployment and Operations" is I think that a common question coming to the docs would be "How do I deploy this?" and I wanted it to be very clear where to go in that case.